deepometry/parse.py

```diff
@@ -52,21 +56,80 @@ def _parse_cif(pathname, output_directory, size, channels):
     for image_index in range(0, image_count, 2):
         image = reader.read(series=image_index)
 
+        if not os.path.exists(output_directory):
+            os.makedirs(output_directory)
+
         parsed_image = numpy.empty((size, size, len(channels)), dtype=numpy.uint8)
 
         for (channel_index, channel) in enumerate(channels):
             parsed_image[:, :, channel_index] = _rescale(_resize(image[:, :, channel], size))
 
         output_pathname = os.path.join(
             output_directory,
-            "{}.npy".format(hashlib.md5(str(time.time()).encode("utf8")).hexdigest())
+            "{}__{}.npy".format(
+                os.path.basename(pathname).replace(".cif", ""),
+                hashlib.md5(str(time.time()).encode("utf8")).hexdigest())
         )
 
         numpy.save(output_pathname, parsed_image)
 
     return True
 
 
+def _parse_tif(src, output_directory, labels, size, channels):
+
+    def channel_regex(channels):
+        return ".*" + "Ch(" + "|".join(str(channel) for channel in channels) + ")"
+
+    regex = channel_regex(channels)
+
+    nested_filenames = []
+
+    for label in labels:
+#        print("Parsing directory: {}".format(label))
+
+        src_dir = os.path.join(src, label)
+
+        filenames = glob.glob("{}/*.tif".format(src_dir))
```

Comment:
Use `os.path.join` to concatenate paths: `os.path.join(src_dir, "*.tif")`

Reply:
I don't need glob.glob when using `os.path.join(src_dir, "*.tif")`, right?